charmcraft/application/commands/init.py

```diff
@@ -58,6 +59,9 @@
         A basic charm but meant to be deployed in machine-based environments,
         without container requirements.
 
+    flask-framework:
+        A basic Flask application charm for the 12-factor charm project.
```

Comment:
```suggestion
        A basic Flask application charm using xiilib.
```

Reply:
Regarding this change, I'm not sure if we should directly inform the user that we're using the `xiilib`, as this is somewhat of an internal implementation detail. I will ask the documentation team about it later, thanks!